sql/dql/basic-selects/select-cols.md

```diff
@@ -28,19 +28,40 @@ aspects:
 ---
 ## Content
 
+Queries in SQL are commonly written to return only the columns of interest. Let's take another look at the `pokemon` table:
+
+| id | name      | type_1 | type_2 | hp | attack |
+|----|-----------|--------|--------|----|--------|
+| 1  | Bulbasaur | Grass  | Poison | 45 | 49     |
+| 2  | Ivysaur   | Grass  | Poison | 60 | 62     |
+| 3  | Venusaur  | Grass  | Poison | 80 | 82     |
+
 If you only want to see *some* of the columns in a table, use a `SELECT` statement followed by a comma-separated list of the columns you wish to retrieve:
 
 ```sql
-SELECT name, height, weight
+SELECT name, hp, attack
 FROM pokemon;
 ```
 
-The above command retrieves the `name`, `height` and `weight` information for all of the *rows* in the `pokemon` table.
+The above command retrieves the `name`, `hp` and `attack` information for all of the *rows* in the `pokemon` table. The resulting table would look something like this:
+
+| name      | hp | attack |
+|-----------|----|--------|
+| Bulbasaur | 45 | 49     |
+| Ivysaur   | 60 | 62     |
+| Venusaur  | 80 | 82     |
+
+Note that every time you run a query, you get a copy of your original table that contains only the requested columns. This resulting table is also commonly referred to as *resulting set*. It is also important to keep in mind that the *result set* isn't stored permanently in the database, and that it doesn't change any tables in the database.
```

Comment:
We should be consistent here and use "result set" for both (not "resulting set")